lib/SauceBrowser.js

```diff
@@ -194,17 +196,19 @@ SauceBrowser.prototype.start = function() {
                             self.debug('res.length: %s', res.length);
 
                             // if we received some data, reset the no output watch timeout
-                            if (res.length > 0) {
+                            if (!_.isEmpty(res)) {
                                 watchOutput();
                             }
 
                             var has_done = false;
                             res = res || [];
-                            res.filter(Boolean).forEach(function(msg) {
+                            _(res).filter(Boolean).forEach(function(msg) {
```

Comment:
why do the lodash wrappers need to be used here? What was wrong with the builtin array functions?